.circleci/config.yml

```diff
@@ -31,6 +31,18 @@ jobs:
       - run: 'go build'
       - run: 'PAYLOAD=`pwd`/mr go test -v'
       - run: 'PAYLOAD=`pwd`/mr go test -bench .'
+  "golang-latest-external-libzstd":
```

Comment:
you can probably update Go 1.14, 1.15 with 1.16 & 1.17 (we just want to make sure we support current go version + previous one)